lib/cashier.rb

```diff
@@ -0,0 +1,20 @@
+require "order_error"
+require "orders"
+require "order"
+
+class Cashier
+
+  def initialize(orders)
+    @orders = orders
+  end
+
+  def take_customers_order(customers_name)
+    if customers_name.nil? || customers_name.empty?
+      raise OrderError.new
+    end
+
+    @orders.add_order(Order.new(customers_name))
```

Comment:
Is it possible for 1 customer to have more than order? If this is the case we might need something more unique than `customers_name` for the order name. If not, we can add a check if a given customer has already placed an order.

Reply:
Agreed, currently it is designed for a single order by name but I have further ideas on having multiple drinks per customer and then having an order ID which would be returned to the customer.